README.md

```diff
@@ -10,6 +10,7 @@ The following are required:
 
 - [Git](https://git-scm.com/)
 - [Node.js](https://nodejs.org/)
+- [Java](https://docs.oracle.com/en/java/javase/11/install/overview-jdk-installation.html#GUID-8677A77F-231A-40F7-98B9-1FD0B48C346A)
```

Comment:
How about this link instead?
https://www.oracle.com/java/technologies/downloads/

Reply:
okay